dependencies.yaml

```diff
@@ -6,7 +6,29 @@ options:
   versionConflictPolicy: highest
 
 dependencies:
+  com.google.dagger:
+    dagger:
+      lang: java
+      modules: [ "", "compiler", "producers", "spi" ]
+      version: "2.26"
+
   com.google.guava:
     guava:
       lang: java
       version: "28.2-jre"
+
+  com.google.protobuf:
+    protobuf:
+      lang: java
+      modules: [ "java", "java-util" ]
+      version: "3.0.0-beta-1"
```

Comment:
this is one of the ones I mentioned in that other commit alas we probably don't want to include here since we depend on this repo via source as well to support protoc/proto_library in bazel working right.

a section at the bottom like:
```

replacements:
  com.google.protobuf:
    protobuf-java:
      lang: java
      target: "@com_google_protobuf//:protobuf_java"
    protobuf-java-util:
      lang: java
      target: "@com_google_protobuf//:protobuf_java_util"
```

will ensure in transitive dependencies from the maven resolve any usages of protobuf get re-mapped onto the from source target